pyirf/tests/test_statistics.py

```diff
@@ -21,3 +21,19 @@ def test_lima_gammapy():
     for n_on, n_off, alpha in zip(n_ons, n_offs, alphas):
         sig_gammapy = WStatCountsStatistic(n_on, n_off, alpha).sqrt_ts
         assert np.isclose(li_ma_significance(n_on, n_off, alpha), sig_gammapy)
+
+
+def test_lima_accuracy():
```

Comment:
Not sure if I get the point of this test.

The aim of this PR is just to use float64 inside this function which means output becomes also float64.

Wouldn't be sufficient to compare the output of the updated function to the same thing done with the gammapy one?

This if we don't want to simply revert to that, approach or which I am in favour anyway.

Reply:
Point was to have a test to play around with until it passes so I converted the "failing" code from #205 into a test. While I'd second that the meaning of this test is somewhat limited I'd like to keep it as a safeguard for prevent that the original problem (float32 and float64 yielding different results) reappears if the function changes.